cli.js

```diff
@@ -148,15 +148,27 @@ async function init() {
 			await execa(path.join(__dirname, 'seticon'), [composedIconPath, dmgPath]);
 
 			ora.text = 'Code signing DMG';
-			let identity;
 			const {stdout} = await execa('/usr/bin/security', ['find-identity', '-v', '-p', 'codesigning']);
-			if (cli.flags.identity && stdout.includes(`"${cli.flags.identity}"`)) {
-				identity = cli.flags.identity;
-			} else if (!cli.flags.identity && stdout.includes('Developer ID Application:')) {
-				identity = 'Developer ID Application';
-			} else if (!cli.flags.identity && stdout.includes('Mac Developer:')) {
-				identity = 'Mac Developer';
-			}
+
+			let identity;
+			stdout.split(/\r?\n/).find(line => {
```

Comment:
Seems like a normal `for-of` loop would be better here as you're not using the return value from `find` anyway.